resdk/analysis/alignment.py

```diff
@@ -1,23 +1,89 @@
-"""Chip Seq analysis."""
+"""Alignment analysis."""
 from __future__ import absolute_import, division, print_function, unicode_literals
 
 from resdk.resources.utils import get_data_id, get_samples
 
 __all__ = ('bowtie2', 'hisat2')
 
 
-def bowtie2(resource, genome):
+def bowtie2(resource, genome, mode=None, speed=None, use_se=None, discordantly=None, rep_se=None,
+            minins=None, maxins=None, trim_5=None, trim_3=None, trim_iter=None, trim_nucl=None,
+            rep_mode=None, k_reports=None):
     """Run bowtie2 aligner on given resource.
 
-    Aligne reads files of given resource to the given genome using the
-    ``bowtie2`` aligner. If reads were already aligned, existing objects
+    Align reads files of given resource to the given genome using the
+    `Bowtie2`_ aligner. If reads were already aligned, existing objects
     will be returned.
 
+    .. _Bowtie2:
+        http://resolwe-bio.readthedocs.io/en/latest/catalog-definitions.html#process-alignment-bowtie2
+
     :param resource: resource of which reads will be aligned
     :param genome: data object with genome that will be used
     :type genome: `~resdk.resources.data.Data`
+    :param str mode: alignment mode (options are: --end-to-end,
+        --local), default is --end-to-end
+    :param str speed: speed vs sensitivity (options are: --very-fast,
+        --fast, --semsitive, --very-sensitive), default is --sensitive
+    :param bool use_se: map as single-ended (for paired-end reads
+        only), default is False
+    :param bool discordantly: report discordantly matched read, default
+        is True
+    :param bool rep_se: report single ended, default is True
+    :param int minins: minimum fragment length, default is 0
+    :param int maxins: maximum fragment length, default is 500
+    :param int trim_5: number of bases to trim from 5', default is 0
+    :param int trim_3: number of bases to trim from 3', default is 0
+    :param int trim_iter: number of iterations, default is 0
+    :param int trim_nucl: number of bases to trim from 3' in each
+        iteration, default is 0
```

Comment:
Default is 2. Please also make sure that all other defaults are accurate.

https://github.com/genialis/resolwe-bio/blob/f99bb013678bca3a6e6e8142967201423f012076/resolwe_bio/processes/alignment/bowtie.yml#L338